### DISCUSSION TITLE
mobx-react package - is deprecated?

### DISCUSSION THREAD

**Discussion by it-efrem** [mobxjs/mobx#2610] [Q&A] [answered]
Hi! mobx-react Now not recommended to use?

https://github.com/mobxjs/mobx-react

![image](https://user-images.githubusercontent.com/20017668/98815627-813d9f80-2438-11eb-94bd-a9cdbcfff263.png)

**Comment by mweststrate** [comment]
We turned it into a mono-repo to make maintenance simpler, so you'll find
mobx-react now here: https://github.com/mobxjs/mobx/tree/main/packages

**Comment by mweststrate** [comment]
For all clarity the package is not deprecated, just this source code
repository to avoid future interactions with it.

**Comment by it-efrem** [comment, marked as answer]
@mweststrate 
I think need update README file, because unclear how now use "react-router"?

in "https://github.com/mobxjs/mobx/tree/main/packages/mobx-react" in string "There is also work-in-progress user guide for additional information" refers to "https://mobx-react.js.org/"

"mobx-react.js.org" it is written there: "This site is DEPRECATED"

how install and use "react-router" now?

**Reply by urugator** [reply]
https://github.com/mobxjs/mobx/pull/2612

**Comment by it-efrem** [comment]
@mweststrate @urugator 
Thanks! Now I will use this approach (https://mobx.js.org/react-integration.html#react-integration) -> ʻuseState` with observable class it is even more convenient!

```
import {observer} from "mobx-react";
import { useState } from "react"
import { makeAutoObservable } from "mobx"

class CounterStore {
  count = 0

  constructor() {
    makeAutoObservable(this)
  }

  increaseCounter() {
    this.count += 1
  }
}

const TimerView = observer(() => {
  const [counterStore] = useState(() => new CounterStore())

  console.log('rerender', counterStore.count)

  return <div onClick={() => counterStore.increaseCounter()}>
    Count: {counterStore.count}
  </div>
})
```

```
    "mobx": "^6.0.3",
    "mobx-react": "^7.0.4",
```
